quickwit/quickwit-actors/src/universe.rs

```diff
@@ -81,17 +95,8 @@ impl Universe {
         self.spawn_ctx.kill_switch.kill();
     }
 
-    /// Simulate advancing the time for unit tests.
-    ///
-    /// This is smarter than just mocking a clock.
-    /// As long as everything is happening in actors handlers, initialize
-    /// or finalize methods, the result should be rigorously the same as
-    /// if we waited.
-    pub async fn simulate_time_shift(&self, duration: Duration) {
-        self.spawn_ctx
-            .scheduler_client
-            .simulate_sleep(duration)
-            .await;
+    pub async fn sleep(&self, duration: Duration) {
```

Comment:
The `simulate_time_shift` seems a better name for this IMHO. We need the bring back the updated docs comment

Reply:
It is not the same semantics as it used to be. it does not accelerate time in itself anymore.

Instead, its semantics is "just sleep".

But the universe can created with an accelerated time.